common/djangoapps/student/tests/test_enrollment.py

```diff
@@ -218,3 +227,126 @@ def _change_enrollment(self, action, course_id=None, email_opt_in=None):
             params['email_opt_in'] = email_opt_in
 
         return self.client.post(reverse('change_enrollment'), params)
+
+    def _enrollment(self):
+        """ Generate a default course mode and enrollment. """
+        mode = CourseMode(
+            course_id=self.course.id,
+            mode_slug='verified',
+            mode_display_name='Verified',
+        )
+        mode.save()
+
+        return CourseEnrollment(
+            mode=CourseMode.VERIFIED,
+            course_id=self.course.id,
+            user=self.user,
+        )
+
+    def test_refundable_can_refund(self):
+        """ Assert that enrollment is refundable if can_refund is set."""
```

Comment:
I agree that `can_refund` is a hack that should be resolved.  I think it is out of scope for this ticket.  I think we should tackle this when we pull the refund logic into Otto.